controllers/rcj_soccer_player_b2/rcj_soccer_player_b2.py

```diff
@@ -0,0 +1,99 @@
+"""rcj_soccer_player controller - ROBOT B2"""
+from controller import Robot
+import struct
+import math
+
+TIME_STEP = 64
+MY_ROBOT_NAME = "B2"
+ROBOT_NAMES = ["B1", "B2", "B3", "Y1", "Y2", "Y3"]
+N_ROBOTS = len(ROBOT_NAMES)
+
+
+def parse_supervisor_msg(packet: str) -> dict:
```

Comment:
@Adman seeing this, two things are obvious:

1. we need to figure out a nice way of sharing code between these players
2. the degree conversion below ought to get extracted in there as well

Let's start a discussion on how to best do that